ui/desktop/electron-app/src/ipc/handlers.js

```diff
@@ -194,6 +194,33 @@ handle('isClientAgentRunning', async () => {
   }
 });
 
+/**
+ * Gets boundary cli version
+ */
+handle('getCliVersion', async () => boundaryCli.version());
+
+/**
+ * Returns desktop version
+ */
+handle('getDesktopVersion', async () => {
+  const desktopVersion = app.getVersion();
+  return desktopVersion;
+});
```

Comment:
```suggestion
handle('getDesktopVersion', async () =>  app.getVersion());
```

Does `app.GetVersion` pull the correct version? We normally pull it from our generated config and the docs says it pulls it from the `package.json` which I don't believe we update.

Reply:
yeah, I wasn't sure which one to use, thought either would work, but good to know we don't keep track of the app one. I will switch this to pull from config